nelc/edx-platform
# feat: New link on studio home to Taxonomy list

## Description
This is a backport of https://github.com/openedx/edx-platform/pull/33439

## Testing instructions

The provided instructions on the original pr are enough, just consider that you can use the branch master of  course-authoring and if you are using a cms custom theme, you should update that
